apps/cnquery/cmd/run.go

```diff
@@ -23,7 +23,7 @@ func init() {
 
 var execCmd = builder.NewProviderCommand(builder.CommandOpts{
 	Use:   "run",
-	Short: "Run an MQL query",
+	Short: "Run an MQL query.",
```

Comment:
Should we really have a dot in that short description. It always confuses me.

Reply:
This was the only one without a period and it looked really off in the main pack docs